.travis.yml

```diff
@@ -18,6 +18,10 @@ go:
 before_install:
   - sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-6 90
   - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-6 90
+  - . $HOME/.nvm/nvm.sh
+  - nvm install 8
+  - nvm use 8
+  - npm install -g yarn
```

Comment:
Aren't we having a `validate and build` stage as in `gitbase-web`?

example:
```yaml
jobs:
  include:
    - stage: validate and build
      script:
        - make test
        - make lint
        - make coverage
        - make validate-commit
```

Reply:
I mentioned it in the PR description:
> we don't have lint stage in CI or lint targets for lookout. We may consider add it later and run web-lint there too.

Reply:
We could include `yarn lint` in the current 'Generated code' travis job. And maybe rename it to 'Validate'

Reply:
my point is we don't have `make lint` target for go code.

If you try to run `golint` on our source code it would produce LOTS of errors.
Also do we still want to run vet? In newer go I believe it works by default.

Or do you think we should add linting only for frontend code?

Reply:
If we enforce it from the beginning for the frontend code, it will be easier to keep it nice and formatted.
To avoid the problem you just mentioned with go, if adding a linter later involves too many changes we won't do it.

Up to you to include it here, or later in another PR after researching more about the better way to do it for typescript. But I would try to include it soon.

Reply:
I'd agree if there is an issue about this, and it is postponed to be done as the next task.